### PR TITLE
Update time.md

### PR DESCRIPTION
**Pull request by almargit**

### CODE REVIEW

File: learn/date/time.md [hunk 1]
```diff
@@ -127,27 +127,111 @@ In this example, `time.time()` is used to retrieve the current time as a floatin
...
+The `timeit` function in Python takes several parameters that allow you to customize its behavior:
+
+- **stmt**: This is the statement you want to measure. It can be a string containing a single statement or multiple statements separated by semicolons.
+- **setup**: This parameter is optional and is used to set up the environment for the code to be measured. It can also be a string containing one or more statements.
+- **timer**: This parameter specifies the timer function to be used. If not specified, the default timer for the current platform will be used.
+- **number**: This parameter determines the number of executions of the code. The more executions, the more accurate the timing measurement.
+
+### Timing Multiple Lines in Python Code
+
+You can use the `timeit` module to time multiple lines of Python code. Here are two examples using different approaches:
```
Comment (angarsky) [inline learn/date/time.md:145]: Let's add a dot after `using different approaches:`. And highlight the "Example" lines as H4 heading.

File: learn/date/time.md [hunk 2]
```diff
@@ -127,27 +127,111 @@ In this example, `time.time()` is used to retrieve the current time as a floatin
...
+
+execution_time = timeit.timeit(stmt=code_to_measure, number=10000)
+print(f"Execution time: {execution_time} seconds")
+```
+
+In both examples, we define a function (`square_numbers`) and then call it using the `timeit` module. The code to be measured is enclosed in triple quotes, allowing us to span multiple lines.
+
+### `timeit()` Methods
+
+The timeit module provides different methods for measuring execution time. Here are some of them:
```
Comment (angarsky) [inline learn/date/time.md:189]: Let's use the dot after `Here are some of them:` + H4 headings.

**Review by angarsky — CHANGES_REQUESTED** [review]
Good job! But let's improve the example headings - see inline comments.

File: learn/date/time.md (hunk 1 above)
Comment (almargit) [reply learn/date/time.md:145]: Done

File: learn/date/time.md (hunk 2 above)
Comment (almargit) [reply learn/date/time.md:189]: Done

**Review by angarsky — APPROVED** [review]
Approved. Thank you!

**Pull request merged**